src/search.cpp

```diff
@@ -469,53 +457,59 @@ void Thread::search() {
       // Do we have time for the next iteration? Can we stop searching now?
       if (Limits.use_time_management())
       {
-          if (!Threads.stop && !Threads.stopOnPonderhit)
```

Comment:
This logic is unneeded IMO. If Threads.stopOnPonderhit is true, the only thing that can happen is that this will be asserted again (except for some inference with doEasyMove, which is less important in light of the other PR). It obviously can be added back, but I don't see the benefit.